examples/web-wagmi/src/inbox/UseConversations.tsx

```diff
@@ -0,0 +1,81 @@
+import { ProfileOwnedByMe, useEnhanceConversations } from '@lens-protocol/react-web';
+import { useClient, useConversations } from '@xmtp/react-sdk';
+import { Link } from 'react-router-dom';
+
+import { LoginButton, WhenLoggedInWithProfile, WhenLoggedOut } from '../components/auth';
+import { Loading } from '../components/loading/Loading';
+import { ConversationCard } from './components/ConversationCard';
+import { EnableConversationsButton } from './components/EnableConversationsButton';
+
+type UseConversationsInnerProps = {
+  profile: ProfileOwnedByMe;
+};
+
+function UseConversationsInner({ profile }: UseConversationsInnerProps) {
+  const {
+    conversations: tempConversations,
+    error: tempError,
+    isLoading: isLoadingTemp,
+  } = useConversations();
+
+  const {
+    data: conversations,
+    error,
+    loading,
+  } = useEnhanceConversations({ conversations: tempConversations, profile });
```

Comment:
wondering if we could model the api in this way:
```
const {} = useEnhanceConversations(useConversations());
```
Than as a consumer you only have to care about single `error/loging/data` variables, what do you think?

Reply:
yes, this is a very good idea! I will try it.